docs/interactions/Receiving_and_Responding.md

```diff
@@ -68,11 +68,12 @@ While the `data` field is guaranteed to be present for all [interaction types](#
-| -------------- | ----------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| custom_id      | string                                                                                                            | the [`custom_id`](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/custom-id) of the component                             |
-| component_type | integer                                                                                                           | the [type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types) of the component             |
-| values?\*      | array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure) | values the user selected in a [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component |
+| Field          | Type                                                                                                                   | Description                                                                                                                                                  |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| custom_id      | string                                                                                                                 | the [`custom_id`](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/custom-id) of the component                                                                          |
+| component_type | integer                                                                                                                | the [type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types) of the component                                                          |
+| values?\*      | array of one of [select menu interaction values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-interaction-values) | values the user selected in a [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component                                              |
+| resolved?      | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)                | details of any users/roles/mentionables/channels selected by the user in a [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component |
 
 \* This is always present for select menu components
 
```

Comment:
```suggestion
| Field          | Type                                                                                                                            | Description                                                                                                                                 |
| -------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
| custom_id      | string                                                                                                                          | [`custom_id`](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/custom-id) of the component                                                             |
| component_type | integer                                                                                                                         | [Type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types) of  component                                                |
| values?\*      | array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure) or snowflakes | Values the user selected in a [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component                             |
| resolved?      | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)                         | Details about any selected resources in an auto-populated [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component |

\* `values` is present for all select menu components. For string select menus (`type` 3), it will be an array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure). For all other select menus, it will be an array of snowflakes corresponding to the resources selected by the user.

```

Reply:
mostly just rewording to account for what you were trying to do in the section on the other page